website/docs/cdktf/create-and-deploy/project-setup.html.md

```diff
@@ -7,66 +7,91 @@ description: "Learn how to build a CDKTF application from a template, project st
+Terraform Cloud supports [connecting to VCS providers](https://www.terraform.io/docs/cloud/vcs/index.html).
+
+To use the VCS integration, commit the synthesized Terraform config (the `cdktf.out` directory) alongside your code so Terraform Cloud can use it to deploy your infrastructure. On the General Settings page of your Terraform Cloud Workspace [set the Terraform Working Directory](https://www.terraform.io/docs/cloud/workspaces/settings.html#terraform-working-directory) to the output directory of the stack you want to deploy. So for example use `cdktf.out/stacks/dev` if your stack is named `dev`.
+
+Note: Be aware, that the synthesized Terraform config might contain credentials or other sensitive data which was provided as input for the `cdktf` application.
 
 #### External CI service
 
-If you prefer to keep the generated artefacts out of your repository, use any CI (Continuous Integration) service to build and deploy them instead. The CDK for Terraform CLI supports deploying to Terraform Cloud using either the local or remote execution mode. For more information on how runs work in Terraform Cloud, see [Terraform Runs and Remote Operations](https://www.terraform.io/docs/cloud/run/index.html).  
+If you prefer to keep the generated artefacts out of your repository, use any CI (Continuous Integration) service to build and deploy them instead. The CDK for Terraform CLI supports deploying to Terraform Cloud using either the local or remote execution mode. For more information on how runs work in Terraform Cloud, see [Terraform Runs and Remote Operations](https://www.terraform.io/docs/cloud/run/index.html).
 In your CI steps use the `cdktf-cli` commands to synthesize your code and deploy your infrastructure:
 
 ```
 cdktf deploy --auto-approve
 ```
 
-
 ## Project Structure
 
 TO DO: Explain how CDKTF projects are structured - What files there are in a project with links to places to find out more information
 
+TO DO: This largely depends on the type of project - Do we really want to list all them here? There are getting started guides which would probably be more appriate https://github.com/hashicorp/terraform-cdk/tree/main/docs/getting-started (sebastian)
+
+The only really relevant file for a user is [cdktf.json](../concepts/fundamentals/cdktf-json.md). This file is used as config file for the cli
 
 ## Convert Existing HCL project
 
 Use the typescript template and add a flag on init to convert from an existing HCL project to TypeScript.
 
-..show example
-
+```
+$ cdktf init --template=typescript --from-terraform-project /path/to/my/tf-hcl-project
+```
 
```

Comment:
The following part should be a section on its own, since it's rather advanced and language specific